dvc/command/diff.py

```diff
@@ -147,15 +31,6 @@ def add_parser(subparsers, parent_parser):
         help=DIFF_HELP,
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
-    diff_parser.add_argument(
-        "-t",
-        "--target",
-        help=(
-            "Source path to a data file or directory. Default None. "
-            "If not specified, compares all files and directories "
-            "that are under DVC control in the current working space."
-        ),
-    )
     diff_parser.add_argument(
```

Comment:
This one is no longer a required one, it should be optional, same as in `dvc metrics diff`.